src/drive/mobile/containers/onboarding/SelectServer.jsx

```diff
@@ -37,39 +64,46 @@ export class SelectServer extends Component {
           >
             <div className={styles['cozy-logo-white']} />
           </div>
-          <input
-            type="url"
-            className={
-              error
-                ? classNames(styles['input'], styles['error'])
-                : styles['input']
-            }
-            placeholder={t(
-              'mobile.onboarding.server_selection.cozy_address_placeholder'
-            )}
-            ref={input => {
-              this.serverInput = input
-            }}
-            onChange={updateServerUrl}
-            value={serverUrl}
-          />
+          <label className={styles['coz-form-label']}>
+            {t('mobile.onboarding.server_selection.label')}
+          </label>
+          <div className={styles['input-partial']}>
+            <span className={styles['prefix']}>https://</span>
+            <input
+              type="text"
+              className={
+                error
+                  ? classNames(styles['input'], styles['error'])
+                  : styles['input']
+              }
+              placeholder={t(
+                'mobile.onboarding.server_selection.cozy_address_placeholder'
+              )}
+              ref={input => {
+                this.serverInput = input
+              }}
+              onChange={updateServerUrl}
+            />
+          </div>
           {!error && (
-            <p className={styles['description']}>
+            <p className={classNames(styles['description'], styles['info'])}>
               {t('mobile.onboarding.server_selection.description')}
             </p>
           )}
           {error && (
-            <p className={styles['description']} style={{ color: 'red' }}>
-              <ReactMarkdown source={t(error)} />
-            </p>
+            <ReactMarkdown
+              className={classNames(styles['description'], styles['error'])}
+              source={t(error)}
+            />
           )}
         </div>
         <footer className={styles['wizard-footer']}>
           <button
             role="button"
             className={classNames(styles['c-btn'], styles['c-btn--regular'])}
-            onClick={() => selectServer(serverUrl)}
+            onClick={() => this.setServerUrl(serverUrl)}
             disabled={error || !serverUrl}
+            aria-busy={fetching ? true : false}
```

Comment:
I first wrote `aria-busy={fetching}` but I wondered if it was explicit enough. You tell me.